0011-interledger-payment-request/0011-interledger-payment-request.md

```diff
@@ -78,6 +82,10 @@ The Crypto Condition to be used as the execution condition for the payment. See
 
 The timestamp when the request expires. Recipients SHOULD NOT fulfill the conditions of incoming transfers that arrive after the expiry has passed. Senders MAY use the request expiry to determine the expiry of the transfer they put on hold for the first connector.
 
+#### data
+
+An arbitrary JSON object that should be included in the ILP Packet's user data header.
```

Comment:
@justmoon What's the current thinking on data headers in the ILP Packet? Would it make sense to have this as JSON and have the sender serialize it into one of the headers or would that be much more complicated than having this encoded in base64 in the payment request?